backend/src/graphql_api/stable_coin.rs

```diff
@@ -425,4 +434,57 @@ impl QueryStableCoins {
         );
         txn_summary
     }
+
+    async fn live_stablecoins(&self, _ctx: &Context<'_>) -> anyhow::Result<Vec<StableCoin>> {
+        let endpoint_env = env::var("CCDSCAN_INDEXER_GRPC_ENDPOINTS");
```

Comment:
We should not reuse the env variables from the indexer
```suggestion
        let endpoint_env = env::var("CCDSCAN_API_GRPC_ENDPOINTS");
```